### PR TITLE
Stringifiers

### PR DESCRIPTION
**Pull request by chmanchester**

### CODE REVIEW

**Comment by chmanchester** [comment]
Let me know if I should split this to a branch per issue. Thanks!

File: tests/wpt/metadata/html/dom/interfaces.html.ini [hunk 1]
```diff
@@ -8658,6 +8658,12 @@
   [Location interface: window.location must inherit property "reload" with the proper type (2)]
     expected: FAIL
 
+  [Location interface: existence and properties of interface prototype object]
+    expected: FAIL
+
+  [Stringification of window.location]
+    expected: FAIL
```
Comment (Manishearth) [inline tests/wpt/metadata/html/dom/interfaces.html.ini]: This seems wrong...

Though it's probably due to lack of support for some other part of the test.

**Comment by Manishearth** [comment]
Nah, keeping it in one PR is fine.

This looks correct (just doublechecked the specs), though it's better if @Ms2ger  or someone reviews it.

File: tests/wpt/metadata/html/dom/interfaces.html.ini (hunk 1 above)
Comment (chmanchester) [inline tests/wpt/metadata/html/dom/interfaces.html.ini]: Yeah, I concluded there was something fishy about that. The second is marked an expected failure in gecko, I'll take another look at the first.

**Comment by jdm** [comment]
\o/

File: tests/wpt/metadata/html/dom/interfaces.html.ini (hunk 1 above)
Comment (chmanchester) [inline tests/wpt/metadata/html/dom/interfaces.html.ini]: Right, both of these tests are expecting an "[object ..." style stringification that's been invalidated.

**Comment by chmanchester** [comment]
@Ms2ger, the issues in critic are resolved. Did I miss a step here, or do you want to do more review?

**Comment by jdm** [comment]
More review is necessary; @Ms2ger is away until Sunday, so someone else may or may not jump on it before then.

**Comment by jdm** [comment]
Ready to squash and merge!

**Pull request closed without merging**